Guidelines.md

```diff
@@ -597,6 +598,51 @@ An API should never trust the input from the client and input SHOULD be validatd
 - Consider logging input validation failures. Assume that someone who is performing hundreds of failed input validations per second is up to no good.
 - Use a secure parser for parsing the incoming messages. If you are using XML, make sure to use a parser that is not vulnerable to XXE and similar attacks.
 
+### 7.14 Logging and audit
+Services compliant with the Digipolis REST API Guidelines MUST write logging to `stdout` and `stderr` according to the [12factor][12factor-logging] guidelines. The format of the logs MUST follow the LogStash JSON standard, one entry per line without zonder `[, ]` or `,`. This is also communicated through the FAST procedure.
```

Comment:
Do we have to mention the FAST procedure here? Shouldn't this document be included as a reference for API guidelines in every call that goes out, including FAST procedures?

Reply:
It's also the only place in the guidelines where we mention that a specific guideline is included in the FAST procedure. So I would leave out the FAST procedure out of the guidelines. I'm not sure if the complete API guidelines will be referenced in a FAST procedure or just a subset? This should be a discussing point on the next API Workgroup if this wasn't mentioned on the previous API Workgroup meetings. I'll check with Jan if  there is any agreement on this subject.